lua/darkrp_modules/levels/sv_addways.lua

```diff
@@ -47,9 +49,13 @@ local time = LevelSystemConfiguration.Timertime
 timer.Create( "PlayXP", time,0,function()
 	if(LevelSystemConfiguration.TimerModule) then
 		for k,v in pairs(player.GetAll()) do 
-			local XP = v:addXP(LevelSystemConfiguration.TimerXPAmount, true)
+			if string.find(v:GetUserGroup(), LevelSystemConfiguration.TimerXPVipGroups) then
```

Comment:
Have you tried this? I think you will get some error about XP being undefined as you are defining it in its own block